Japanese/JapaneseInstaller.iss

```diff
@@ -0,0 +1,29 @@
+[CustomMessages]
```

Comment:
Inno Setup requires localization files to be saved using **UTF-8 with BOM**.
Could you please add BOM to this file, so I can merge this?
There is no need to add BOM to other files.